mirdata/acousticbrainz_genre.py

```diff
@@ -0,0 +1,3800 @@
+suited for evaluation of hierarchical multi-label genre classification systems.
+
+Description about the music features can be found here: https://essentia.upf.edu/streaming_extractor_music.html
+
+The datasets are used within the MediaEval AcousticBrainz Genre Task. The task is focused on content-based music
+genre recognition using genre annotations from multiple sources and large-scale music features data available in the
+AcousticBrainz database. The goal of our task is to explore how the same music pieces can be annotated differently by
+different communities following different genre taxonomies, and how this should be addressed by content-based genre r
+ecognition systems.
+
+We provide four datasets containing genre and subgenre annotations extracted from four different online metadata sources:
+
+AllMusic and Discogs are based on editorial metadata databases maintained by music experts and enthusiasts. These sources contain explicit genre/subgenre annotations of music releases (albums) following a predefined genre namespace and taxonomy. We propagated release-level annotations to recordings (tracks) in AcousticBrainz to build the datasets.
+
+Lastfm and Tagtraum are based on collaborative music tagging platforms with large amounts of genre labels provided by their users for music recordings (tracks). We have automatically inferred a genre/subgenre taxonomy and annotations from these labels.
+
+For details on format and contents, please refer to the data webpage.
+
+Note, that the AllMusic ground-truth annotations are distributed separately at https://zenodo.org/record/2554044.
+
+
+
+Citation
+
+If you use the MediaEval AcousticBrainz Genre dataset or part of it, please cite our ISMIR 2019 overview paper:
+
+Bogdanov, D., Porter A., Schreiber H., Urbano J., & Oramas S. (2019).
+The AcousticBrainz Genre Dataset: Multi-Source, Multi-Level, Multi-Label, and Large-Scale.
+20th International Society for Music Information Retrieval Conference (ISMIR 2019).
+
+
+Acknowledgements
+
+This work is partially supported by the European Union’s Horizon 2020 research and innovation programme under grant agreement No 688382 AudioCommons.
+"""
+
+import csv
+import json
+
+import librosa
+import numpy as np
+import os
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import track
+from mirdata import utils
+
+DATASET_DIR = 'acousticbrainz_genre'
+REMOTES = {
+    'annotations': download_utils.RemoteFileMetadata(
+        filename='The AcousticBrainz Dataset Annotations.tar.gz',
+        url='http://isophonics.net/files/annotations/The%20AcousticBrainz Dataset%20Annotations.tar.gz',
+        checksum='62425c552d37c6bb655a78e4603828cc',
+        destination_dir='annotations',
+    )
+}
+
+DATA = utils.LargeData('acousticbrainz_genre_index.json')
+
+
+class Track(track.Track):
+    """AcousticBrainz Dataset track class
```

Comment:
Please be careful about the naming of this, it's _not_ a general AcousticBrainz dataset.

Reply:
nit done